test/jdk/javax/swing/JSlider/bug6587742.java

```diff
@@ -24,26 +24,62 @@
 /* @test
  * @bug 6587742
  * @summary filling half of a JSlider's track is no longer optional
- * @author Pavel Porvatov
- * @run applet/manual=done bug6587742.html
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual bug6587742
  */
```

Comment:
Can be moved to the bug6587742 class declaration for better readability

Reply:
There is a discussion going on for "where to put jtreg tags ?". Since there was no final conclusion, I left it as it was before the conversion.

Updated now.